src/pages/faq.tsx

```diff
@@ -0,0 +1,68 @@
+import { QuestionCard } from "@/components/cards/question-card";
+import { IconPencil } from "@tabler/icons-react";
+import Head from "next/head";
+
+const faq = [
+  {
+    question: "Can I edit my saves with stardew.app?",
+    answer:
+      "stardew.app is a perfection checklist, not a save file editor. While we allow you to upload your save files to the site, we do not store any more data than is needed to update you on your perfection status. We do not store your save file after upload and unfortunately cannot retrieve it for you.",
+    icon: IconPencil,
+  },
+  {
+    question: "How do I upload my save file?",
+    answer:
+      "To upload your save file, click on the 'Upload Save' button on the homepage. You will be prompted to select your save file from your computer. Once you have selected your save file, you will be able to see your progress on the homepage.",
+    icon: IconPencil,
+  },
+  {
+    question: "Can I track multiple save files?",
+    answer:
+      "Yes! You can track multiple save files by uploading each save file separately. You can switch between save files by clicking on the 'Switch Save' button on the homepage.",
+    icon: IconPencil,
+  },
+];
+
+export default function FAQ() {
+  return (
+    <>
+      <Head>
```

Comment:
This probably isn't the right PR, but this feels like it should be ripped into a template component at some point, probably through the <main> component declaration.